bin/git/git.js

```diff
@@ -63,6 +63,21 @@ const TesterHasBuildsWithCommitsQuery = gql`
   }
 `;
 
+const TesterBaselineCommitsQuery = gql`
+  query TesterBaselineCommitsQuery($branch: String!, $parentCommits: [String!]!) {
+    app {
+      baselineBuilds(branch: $branch, parentCommits: $parentCommits) {
+        number
+        status
```

Comment:
Needs `legacy:false`!!

Reply:
That is such a gotcha 🤦‍♂️

Reply:
IKR. I'm not sure what we can do about it, beyond renaming the field entirely.